test/src/databases/all/expr.spec.ts

```diff
@@ -388,12 +388,12 @@ describe.each(runtimes.runtimeList)('%s', (databaseName, runtime) => {
     `).malloyResultMatches(expressionModel, {a: 312});
   });
 
-  test.when(!['postgres', 'snowflake'].includes(runtime.connection.name))(
+  test.when(!['postgres'].includes(runtime.connection.name))(
     'sql safe cast',
     async () => {
       await expect(`
       run: ${databaseName}.sql('SELECT 1 as one') -> { select:
-        bad_date is '123':::date
+        bad_date is '12a':::date
```

Comment:
originally in executor we set QUOTED_IDENTIFIERS_IGNORE_CASE =  TRUE, so we always ignore case even if quoted, so whether we accessed it as flight."id" or flight.id, we were able to read from the id/ID field.

here I am just asserting that QUOTED_IDENTIFIERS_IGNORE_CASE = FALSE, which is the behavior we finalized for malloy.

what I found is that if I do not set this to FALSE explicitly, for some reason output would look like
BAD_DATE, BAD_NUMBER, GOOD_NUMBER, though in generated SQL we have "bad_date", "bad_number", "good_number", etc.

I also ran tests under databases/all and all succeeded for me:

```
 MALLOY_DATABASE=snowflake npx jest --runInBand test/src/databases/all
 
Test Suites: 10 passed, 10 total
Tests:       15 skipped, 354 passed, 369 total
Snapshots:   0 total
Time:        397.491 s
Ran all test suites matching /test\/src\/databases\/all/i.
 ```